package/setup.py

```diff
@@ -597,10 +597,10 @@ def long_description(readme):
                        'trajectories generated by CHARMM, Gromacs, NAMD, LAMMPS, or Amber.'),
           long_description=LONG_DESCRIPTION,
           long_description_content_type='text/x-rst',
-          author='Naveen Michaud-Agrawal',
-          author_email='naveen.michaudagrawal@gmail.com',
+          author='MDAnalysis Development Team',
+          author_email='mdanalysis@numfocus.org',
```

Comment:
Are we happy with _MDAnalysis Development Team_ as author?

Reply:
Yup, I'm happy with it. I'll let @richardjgowers / @IAlibay / @nmichaud approve the PR though.